### ISSUE TITLE
S

### ISSUE THREAD

**Issue by ishandutta2007** [InstaPy/InstaPy#4427]
p

**Comment by timgrossmann** [comment]
@ishandutta2007 Yeah, problem is that that really is their latest version: https://github.com/Clarifai/clarifai-python
So we probably have to propose a PR with the changes to the API repository.

**Comment by WSLUser** [comment]
Related: I get `ERROR: requests 2.21.0 has requirement urllib3<1.25,>=1.21.1, but you'll have urllib3 1.25.2 which is incompatible.`

**Comment by notvexxed** [comment]
I've been experiencing the same issues.

**Comment by jdshaeffer** [comment]
Is there a fix for this right now? I followed all the steps for the Ubuntu installation and I'm experiencing that same error that @ishandutta2007 is getting, as well as a 
### ERROR: launchpadlib 1.10.6 requires testresources, which is not installed.

**Comment by ksiendzu** [comment]
Hello guys!
After 3 hours of fighting with errors during setting up instapy on fedora while getting errors i finally got it. Instapy is running despite jsonschema error and is running on firefox instead of chrome. To be honest I dont know exactly what was wrong and why it finally worked out, so I'll just leave exact lines i typed in fedora terminal and leave it up to you if you're interested to find out which lines are necessary to make it happen:

pip install instapy
sudo yum install gcc
sudo sh install.sh
pip install jsonschema
gem install json-schema-2.5.0.gem
sudo pip install jsonschema==2.6.0
sudo sh install.sh
sudo yum install apt
sudo sh install.sh
sudo pip install instapy --ignore-installed
pip install instapy -U
sudo yum install python-devel
pip install instapy -U

//after line down below there's no more "command 'gcc' failed with exit status 1" error
sudo dnf install python3-devel
sudo pip install instapy -U

//after line below only error is 'clarifai 2.6.1 has requirement jsonschema<3,>=2.5, but you'll have jsonschema 3.0.1 which is incompatible'
sudo pip install instapy --ignore-installed
sudo sh install.sh

//now file quickstart.py is executing
python3 quickstart.py

//lines I entered to switch from chrome to mozilla firefox
sudo dnf config-manager --set-enabled google-chrome
python3 quickstart.py
// after that I got error about 'No such file or directory geckodriver' so just google how to add that to path. I used this link: https://stackoverflow.com/questions/40388503/how-to-put-geckodriver-into-path/40392714
I get troubles with copying unziped file due to permision denied, so i copied it using terminal

sudo cp {drag and drop geckodriver file to get its source} /usr/local/bin/

//In the quickstart.py file I changed line 'headless_browser=False' to 'use_firefox=True' and typed in username and password in proper lines

python3 quickstart.py

And then instapy worked on my fedora 25 using firefox instead of chrome. 
Hope it will help any of you!
